### ISSUE TITLE
Question re motor

### ISSUE THREAD

**Issue by bancroftd** [TobiasWeis/robotcontrol-javascript#2]
Hi Tobias

Any chance I could ask you if for one of the joy sticks its possible to drive a motor forward and reverse. Also the servo rest position, can this be made to be center? I am trying to make an aquatic car. SO I am going to put a FPV camera and stream to mobile and change the buttons to lift the wheels.

**Comment by TobiasWeis** [comment]
Hi, there is already a Servo-class in the robot-folder, the servo should center if you let go of the respective joystick. In a similar manner to the Servo-Class, you could add a DC-Motor-class and just change the mapping of joystick-value to GPIO-PWM-output.

**Comment by michielVanDerEecken** [comment]
First of all, thank you for this awesome code. I've got a question concerning this resolution. I've created a copy of the Servo.py  to Motor.py and then added this in main.py and changed the GPIO mapping
![image](https://user-images.githubusercontent.com/6802642/99876247-d5ddd900-2bf5-11eb-92fc-41f52051e644.png)

However, i'm not able to set my servo using this class, am I forgetting something?

kind regards.
